config.js

```diff
@@ -10,18 +10,30 @@
  ***************************************************/
 
 // Your Firebase instance where we will listen and write search results
-exports.FB_URL   = 'https://' + process.env.FB_NAME + '.firebaseio.com/';
-
-// Either your Firebase secret or a token you create with no expiry, used to authenticate
-// To Firebase and access search data.
-exports.FB_TOKEN = process.env.FB_TOKEN || null;
+exports.FB_URL   = 'https://' + process.env.FB_NAME + '.firebaseio.com';
 
 // The path in your Firebase where clients will write search requests
 exports.FB_REQ   = process.env.FB_REQ || 'search/request';
 
 // The path in your Firebase where this app will write the results
 exports.FB_RES   = process.env.FB_RES || 'search/response';
 
+// See https://firebase.google.com/docs/web/setup#project_setup for how to
```

Comment:
The appropriate doc link here would be https://firebase.google.com/docs/server/setup.